src/DevServer.php

```diff
@@ -37,18 +42,19 @@ class DevServer
 	private $httpClient;
 
 
-	public function __construct(bool $enabled, string $url, float $timeout, ClientInterface $httpClient)
+	public function __construct(bool $enabled, string $url, string $proxy, float $timeout, ClientInterface $httpClient)
 	{
 		$this->enabled = $enabled;
 		$this->url = $url;
+		$this->proxy = $proxy;
 		$this->timeout = $timeout;
 		$this->httpClient = $httpClient;
 	}
 
 
 	public function getUrl(): string
 	{
-		return $this->url;
+		return $this->proxy ? $this->proxy : $this->url;
```

Comment:
I'd make `$proxy` nullable and use `??` here (rather than in the extension), it seems more explicit.

Reply:
not sure about this, never seen or use it before